vcbuild.bat

```diff
@@ -530,7 +530,7 @@ if not defined NODE_VERSION (
 if not defined DISTTYPE set DISTTYPE=release
 if "%DISTTYPE%"=="release" (
   set FULLVERSION=%NODE_VERSION%
-  exit /b 0
+  goto distexit
```

Comment:
I think at some point the subroutine was used more than once... Now it's a degenerate label, but I as well would rather keep it this way...